src/OpenSage.FileFormats.Big/BigArchive.cs

```diff
@@ -47,7 +47,11 @@ public BigArchive(string filePath, BigArchiveMode mode = BigArchiveMode.Update)
                 fileAccess,
                 fileShare));
 
-            Read();
+            // Read if the archive already exists
+            if(_stream.Length > 0 &&  mode != BigArchiveMode.Create)
```

Comment:
Why do we need the check for `_stream.Length > 0`?

Reply:
We always read the big file header, but i should probably add that check somewhere else